src/lubed/obs.py

```diff
@@ -148,11 +146,11 @@ def _query_package(
     package: Package,
     credentials: OBSCredentials,
     api_url: str,
-) -> str:
+) -> (str, bool):
```

Comment:
This change requires an update of the docstring.

Reply:
No docstring exists here (a private method)

Reply:
Fine for me, it's still small enough to see what's going on without a docstring.